release-notes.md

```diff
@@ -179,7 +183,6 @@
 ## O2Saturation (zib-O2Saturation, nl-core-O2Saturation)
 * `.effective[x]:effectiveDateTime` ; mapping ; Renamed mapping from O2SaturationDateTime to SpO2SaturationDateTime. ; ZIB-1217
 * `.bodySite` ; mapping ; Added mapping of MeasuringLocation. ; ZIB-1217
-* `.component:extraOxygenAdministration.code` ; textual ; Removed guidance for ExtraOxygenAdministration. ; ZIB-1217, ZIB-1476
```

Comment:
Why is this line removed?

Reply:
After consulting with @ArdonToonstra , we decided to remove the line. For the implementer, the comment adds nothing.

Reply:
Ok